config/application.rb

```diff
@@ -19,7 +24,8 @@ class Application < Rails::Application
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
 
-    config.action_mailer.preview_path ||= defined?(Rails.root) ? Rails.root.join("lib", "mailers", "previews") : nil
+    config.action_mailer.preview_paths << (defined?(Rails.root) ? Rails.root.join("lib", "mailers", "previews") : nil)
```

Comment:
I'm wondering if the difference between `||=` and `<<` is what's causing the failing rspec test in github @sarvaiyanidhi

Reply:
In Rails 7.1.0 there is now support for multiple preview paths so I am appending as per new configuration.

https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#support-multiple-preview-paths-for-actionmailer-preview

In one of the earlier commit, I still reverted to earlier code base to check if specs pass with it but encountered same issue so not sure if this is the problem https://github.com/rubyforgood/casa/pull/5478/commits/1d639dbacbf08e3b0c70bb5149dbd53b21fc3c5a